### ISSUE TITLE
[Request] Add damage bypasses to the trait advancement

### ISSUE THREAD

**Issue by Korghen** [foundryvtt/dnd5e#2661]
It could be nice to be able to get damage bypasses into the trait advancement.
I have some subclasses that allow non magical damage at a point.

**Comment by Zanderaf** [comment]
Right, I think War Domain cleric has physical resistance with magical bybass
